librocksdb_sys/Cargo.toml

```diff
@@ -23,27 +26,17 @@ sse = ["libtitan_sys/sse"]
 [build-dependencies]
 cc = "1.0.3"
 cmake = "0.1"
-bindgen = "0.51"
+bindgen = "0.53.2"
```

Comment:
Let's don't update bindgen for now. Got conflict somehow recently. rest LGTM.

Reply:
okay